src/Discord/Parts/User/Member.php

```diff
@@ -166,6 +166,23 @@ public function removeRole($role): ExtendedPromiseInterface
 
         return \React\Promise\reject(new \Exception('User does not have role.'));
     }
+    
+    /**		
+    * Sends a message to the user.
+    *
+    * @param string     $message The text to send in the message.
+    * @param bool       $tts     Whether the message should be sent with text to speech enabled.
+    * @param Embed|null $embed   An embed to send.
+    *
+    * @return ExtendedPromiseInterface
+    * @throws \Exception
+    */
+    public function sendMessage(string $message, bool $tts = false, ?Embed $embed = null): ExtendedPromiseInterface
+    {
+        return $this->getUserAttribute()->getPrivateChannel()->then(function ($channel) use ($message, $tts, $embed) {
+            return $channel->sendMessage($message, $tts, $embed);
+        });
```

Comment:
```suggestion
        return $this->user->sendMessage($message, $tts, $embed);
```
Just for better code reuse.

Reply:
LGTM!